src/VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -19,6 +19,13 @@
         //Relative or absolute file system path  where platform will discover installed modules
         "DiscoveryPath": "./Modules"
     },
+    "Auth": {
+        "Authority": "http://localhost:10645/",
```

Comment:
Left Authority is empty and the following comment 
```XML
auth server's url, e.g. http://localhost:5051/ or https://auth.example.com/ or leave empty for mode when authorization
```
// and resource server is the same application